test/DocumentFormat.OpenXml.Tests/CreateFromTemplateTests.cs

```diff
@@ -20,7 +20,7 @@ public void CanCreatePresentationFromTemplate()
                 var part = packageDocument.PresentationPart;
                 var root = part.Presentation;
 
-                packageDocument.SaveAs(Path.GetTempFileName()).Close();
+                packageDocument.SaveAs(Path.GetTempFileName()).Dispose();
```

Comment:
Can you refactor these usages to be within a using block?

Reply:
yes, of course